extensions/amp-analytics/0.1/vendors.js

```diff
@@ -192,6 +192,34 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
     },
   },
 
+  'umenganalytics': {
+    'vars': {
+      'siteid': '',
+      'initial_view_time': '',
+      'eventName': '',
+      'eventProps': '',
+    },
+    'requests': {
+      'base': 'https://b.cnzz.com/utrack?' +
+        '&_siteid=${siteid}' +
+        '&_distinct_id=${clientId(umeng_amp_id)}' +
+        '&_t=${timestamp}' +
+        '&_s=google' +
+        '&_b=web' +
+        '&_r=${externalReferrer}' +
+        '&_h=${screenHeight}' +
+        '&_w=${screenWidth}' +
+        '&_ivt=${initial_view_time}',
+      'pageview': '${base}&_ename=$w_page_view&_eprops=${eventProps}',
+      'event': '${base}&_ename=${eventName}&_eprops=${eventProps}',
+    },
+    'transport': {
+      'beacon': false,
```

Comment:
To confirm that your server only support transport type `image`. We recommend using `beacon`, and AMP will handle switching to other transport method when browser doesn't support `beacon`.

Reply:
We do not support beacon now and will support it later.